Apps/AdminWebClient/src/ClientApp/src/services/restEmailAdminService.ts

```diff
@@ -30,13 +30,16 @@ export class RestEmailAdminService implements IEmailAdminService {
 
   public resendEmails(emailIds: string[]): Promise<string[]> {
     return new Promise((resolve, reject) => {
+      let headers: Dictionary<string> = {};
+      headers["Content-Type"] = "application/json; charset=utf-8";
       this.http
         .post<RequestResult<string[]>>(
           `${this.BASE_URI}`,
-          JSON.stringify(emailIds)
+          JSON.stringify(emailIds),
+          headers
         )
         .then(requestResult => {
-          this.handleResult(requestResult, resolve, reject);
```

Comment:
looks odd, this function does not return anything?

Reply:
It does not, result is based on the request status (200, 400 etc)